pkg/pfcpsim/pfcpsim.go

```diff
@@ -68,7 +66,11 @@ func (c *PFCPClient) getNextSequenceNumber() uint32 {
 }
 
 func (c *PFCPClient) getNextFSEID() uint64 {
+	c.seqNumLock.Lock()
```

Comment:
We shouldn't use the same lock as for sequence number. Since this is a simulator it's ok to have a single lock per client, but please give it more generic name.

Reply:
Right. I added a new lock